wiki/History_of_osu!/1/en.md

```diff
@@ -9,39 +9,54 @@ The following are some of the history of osu! since its beginning. Special thank
+![osu!editor](img/200707-05.jpg "osu!editor")
+![Song Selection for "Edit Mode"](img/200707-06.jpg "Song Selection for \"Edit Mode\"")
+![Slider creation at osu!editor](img/200707-07.jpg "Slider creation at osu!editor")
+![Gameplay example](img/200707-08.jpg "Gameplay example")
 
 August
 ------
 
 **The main menu was updated with a new look**, and the **editor received a lot of new functionality, including copy/paste support**. The Windows's menubar and cursor are now present, making editor features a lot more accessible than before. **Sliders are more extensively supported**, with ticks present and disappearing as you pass them. **Stacking of hitcircles is now also present. Local high scores are a lot better supported. The .osu file format is revised and completely revamped, but includes conversion of older files to the new format.**
 
-<File:200708-01.jpg%7CSong> Selection screen for "Play Mode" <File:200708-02.jpg%7CEvolution> of hit-circle score bursts.
+![Song Selection screen for "Play Mode"](img/200708-01.jpg "Song Selection screen for \"Play Mode\"")
+![Evolution of hit-circle score bursts.](img/200708-02.jpg "Evolution of hit-circle score bursts.")
 
 Gameplay was enhanced with the **debut of combo fire, the HP bar, mouse trails, and better looking sliders**. By this point, we had a humble collection of around **10 beatmaps made by the testing team of around 5 people.**
 
-<File:200708-03.jpg>|"Edit Mode" song selection <File:200708-04.jpg%7CGameplay> example
+!["Edit Mode" song selection"](img/200708-03.jpg "\"Edit Mode\" song selection")
+![Gameplay example](img/200708-04.jpg "Gameplay example")
 
 September
 ---------
 
-<img src="200709-Release.png" title="fig:200709-Release.png" alt="200709-Release.png" width="500" />
+![200709-Release.png](img/200709-Release.png "200709-Release.png")
+
 **The birth of osu!** as far as the public is concerned. The **forums** as we know them today were **established**, and hosted from peppy's home server (ppy.sh). Attention was gained via a news announcement on <http://www.bemanistyle.com> which drew quite a few interested people with previous rhythm game experience.
 
-<File:200709-01.jpg%7CSlider> creation at osu!editor <File:200709-02.jpg%7CMain> Menu <File:200709-03.jpg%7CSpinner> (based on DS's official Ouendan) <File:200709-04.jpg%7CGrade> screen <File:200709-SS.jpg%7CNews> announcement on a gaming website.
+![Slider creation at osu!editor](img/200709-01.jpg "Slider creation at osu!editor")
+![Main Menu](img/200709-02.jpg "Main Menu")
+![Spinner(based on DS's official Ouendan)](img/200709-03.jpg "Spinner(based on DS's official Ouendan)")
+![Grade screen](img/200709-04.jpg "Grade screen")
+![News announcement on a gaming website.](img/200709-SS.jpg "News announcement on a gaming website.")
 
-<File:Easy_v1.png>|[Easy](EZ "wikilink") mod <File:No_Fail_v1.png>|[No Fail](No_Fail "wikilink") mod
+#### Game Modifiers introduced
+![Easy Mod](img/Easy_v1.png "Easy Mod")
```

Comment:
might as well use the full size one
![easy_v1](https://cloud.githubusercontent.com/assets/14264992/25783253/a7ec1a58-331e-11e7-9b04-9f2b93263dfb.png)